sdcm/provision/common/utils.py

```diff
@@ -135,16 +135,16 @@ def install_syslogng_service():
         elif apt-get --help 2>/dev/null 1>&2 ; then
             if dpkg-query --show syslog-ng ; then
                 rm /etc/syslog-ng/syslog-ng.conf  # Make sure we have default syslog-ng.conf
-                apt-get purge -y syslog-ng*
-                DPKG_FORCE=confmiss apt-get --reinstall -y install syslog-ng
+                apt-get purge -o DPkg::Lock::Timeout=300 -y syslog-ng*
+                DPKG_FORCE=confmiss apt-get --reinstall -o DPkg::Lock::Timeout=300 -y install syslog-ng
                 SYSLOG_NG_INSTALLED=1
             else
                 cat /etc/apt/sources.list
                 for n in 1 2 3 4 5 6 7 8 9; do # cloud-init is running it with set +o braceexpand
-                    if apt-get -y update 2>&1 | tee /tmp/syslog_ng_install.output || grep NO_PUBKEY \
-        /tmp/syslog_ng_install.output; then
```

Comment:
@fruch do you have idea why it was done this way?

Reply:
I can only guess, someone was trying to ignore specific errors

No have the key configured correctly, properly was quite common at some point

But regardless it was wrong, and broken

It does point we might still want some kind of lock on this logging setup, and the done file isn't completely enough